docs/index.md

```diff
@@ -107,23 +107,28 @@ This framework defines a general approach to **representing demographic and clin
 
 ## Generating and resolving cryptographic keys
 
-The following key types are used in the Health Cards Framework, represented as JSON Web Keys (see [RFC 7517](https://tools.ietf.org/html/rfc7517)):
+The following key types are used in the Health Cards Framework:
+* Elliptic Curve keys using the P-256 curve
 
-* **Signing Keys**
-    * SHALL have `"kty": "EC"`, `"use": "sig"`, and `"alg": "ES256"`
-    * SHALL have `"kid"` equal to the base64url-encoded SHA-256 JWK Thumbprint of the key (see [RFC7638](https://tools.ietf.org/html/rfc7638))
-    * Signing *Health Cards*
-        * Issuers sign Health Card VCs (Verifiable Credentials) with a signing key (private key)
-        * Issuer publish their signing keys (public key) at `/.well-known/jwks.json`
-        * Wallets and Verifiers validate Issuer signatures on Health Cards
+### Signing *Health Cards*
+* Issuers sign Health Card VCs (Verifiable Credentials) with a signing key (private key)
+* Issuer publish the corresponding public key (public key) at `/.well-known/jwks.json`
+* Wallets and Verifiers use the public key to verify Issuer signatures on Health Cards
 
 ### Determining keys associated with an issuer
+Each public key used to verify signatures is represented as a JSON Web Key (see [RFC 7517](https://tools.ietf.org/html/rfc7517)):
+* SHALL have `"kty": "EC"`, `"use": "sig"`, and `"alg": "ES256"`
+* SHALL have `"kid"` equal to the base64url-encoded SHA-256 JWK Thumbprint of the key (see [RFC7638](https://tools.ietf.org/html/rfc7638))
+* SHALL have `"crv": "P-256`, and `"x"`, `"y"` equal to the base64url-encoded values for the public Elliptic Curve point coordinates (see [RFC7518](https://tools.ietf.org/html/rfc7518#section-6.2))
+* SHALL NOT have the Elliptic Curve private key parameter `"d"`
+* If the issuer has an X.509 certificate for the public key, SHALL have `"x5c"` equal to an array of one or more base64-encoded (not base64url-encoded) DER representations of the public
```

Comment:
Is it possible for an issuer to belong to >1 trust framework? And if so, is there a standard way to publish >1 chain? Does this matter?

Reply:
Yes, to publish >1 chain you can duplicate the JWK entry with all parameters the same except the x5c. I considered including that, but decided to lean towards 'less is more', at least to start. As far as does this matter, hard to say, but there is a solution.  I'll stick in a short sentence, though, since others may ask the same question...

Reply:
Perfect -- it's a solution whose clunkiness is a good match for the [YAGNI](https://en.wikipedia.org/wiki/You_aren't_gonna_need_it)-ness of the request :)

Reply:
Any thoughts about using x5u (URLs) vs. integrating the chain directly in the issuer key set? This could simplify the multi-framework situations. Granted, we would need another cert chain resolution mechanism, but this flexibility might be useful for some applications (e.g., using a trusted key repo)

Reply:
ok I added in the JWK details section but held off on adding the reciprocal information in the validation section based on above.